doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1415,7 +1417,19 @@ We are absolutely aware that it's confusing to have both ``dataverse.fqdn`` and
 dataverse.files.directory
 +++++++++++++++++++++++++
 
-This is how you configure the path Dataverse uses for temporary files. (File store specific dataverse.files.\<id\>.directory options set the permanent data storage locations.)
+Please provide an absolute path to a directory backed by some mounted file system. This directory is used for a number
+of purposes:
+
+1. ``<dataverse.files.directory>/temp`` after uploading, data is temporarily stored here for ingest and/or before
+   shipping to the final storage destination.
+2. ``<dataverse.files.directory>/sword`` a place to store uploads via the :doc:`../api/sword` before transfer
+   to final storage location and/or ingest.
+3. ``<dataverse.files.directory>/<PID Authority>/<PID Identifier>`` data location for file system imports, see
```

Comment:
Hmm - has this been tested? Just wondering if the store uses a different dir whether this is still the right place, or if they have to go in the dir for the store being used. From the code it looks like that is the case. If so, the effect here is that #3 should be dropped (it would just be another case for line 1431) but we should either fix or add an issue for changing the documentation for import.

Reply:
@poikilotherm and I talked this out and he will rewrite this.